docs/elastic_inference.md

```diff
@@ -0,0 +1,182 @@
+# Model Serving with Amazon Elastic Inference 
+
+## Contents of this Document
+* [Introduction](#introduction)
+* [Custom Service](#custom-service)
+* [Creating a EC2 instance with EIA support](#creating-a-ec2-instance-with-eia-support)
+* [Custom Service file with EIA](#custom-service-file-with-eia)
+* [Running elastic inference on a resnet-152](#running-elastic-inference-on-a-resnet-152)
+## Introduction
+
+Amazon Elastic Inference allows you to attach low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost of running deep learning inference by up to 75%. With MMS it is easy to deploy an MXNet based model, taking advantage of the attacble hardware accelerator.
+In this document we explore using Elastic Inference Acclerator (EIA) attached to a Compute Optimized EC2 instance.
+
+## Custom Service
+
+The capability to run model inference with the Elastic Inference Accelerator (EIA) can be achieved by building the custom service to use the EIA context rather than a GPU or CPU context. An MXNet version with support for EIA is required.
+
+To understand the basics of writing a custom service file refer to the [Custom Service Documentation](https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md).
+
+## Creating a EC2 instance with EIA support 
+
+To Create an EC2 instance with EIA support there are few pre-requisites. These include:
+1. Configuring a Secuirty Group
+2. Creating a secure Endpoint.
+3. Creating a IAM role policy.
+    
+The above steps are explored in detail in [AWS Elastic Inference documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-ei.html)  
+
+On completing the above steps, following steps need to be followed in launching a instance with EIA support
+
+1. Open the Amazon EC2 console at https://console.aws.amazon.com/ec2/.
+
+2. Choose Launch Instance.
+
+3. Under Choose one of the Deep Learning AMIs, we recommend Deep Learning AMI v20. This is required to use MXNet with EIA.
```

Comment:
Why not say v20 or later? DL AMI releases a new version every month or so, and you will probably not update this page every DL AMI release.